provd/operation.py

```diff
@@ -44,7 +43,7 @@ def __init__(self, label=None, state=OIP_WAITING, current=None, end=None, sub_oi
 
 
 def format_oip(oip):
-    """Format an operation in progress to a string.
+    r"""Format an operation in progress to a string.
```

Comment:
Is it really necessary?

Reply:
In theory I think you are supposed to use raw strings when you have escape characters. This one might be ok, but the next one has `\w` so it definitely should.

Reply:
ok